lib/ReactViews/Map/Panels/SharePanel/ShareUrl/ShareUrlClipboard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+import Clipboard from "../../../../Clipboard";
+import { ShareUrlInput } from "./ShareUrlInput";
+import Terria from "../../../../../Models/Terria";
+import {
+  Category,
+  ShareAction
+} from "../../../../../Core/AnalyticEvents/analyticEvents";
+
+interface IShareUrlClipboardProps {
+  terria: Terria;
+  theme: "dark" | "light";
+  inputTheme?: "dark" | "light";
+  rounded: boolean;
+  shareUrl: string;
+  placeholder?: string;
+}
+
+export const ShareUrlClipboard: FC<IShareUrlClipboardProps> = ({
```

Comment:
Would we ever use this component outside `ShareUrl` component? Could it go back inside there to reduce modularisation a little bit?